src/app.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const path = require('path');
+
+const app = express();
+
```

Comment:
Mohanned , please add comment to the midleware, handle bare sections in the server .

Reply:
Ok, make it by yourself